frontend/tests/e2e-tests.sh

```
@@ -167,6 +171,49 @@
+
+run_tests() {
+  if [[ -n "$ENTERPRISE" ]]; then
+    echo "Running tests for the enterprise version..."
+    cd "$APP_DIR"/enterprise/frontend || exit 1
+    make pre-tests
+    cd "$APP_DIR"/enterprise/frontend/.build/frontend || exit 1
+  else
+    echo "Running tests for the community version..."
+  fi
+  if [[ -n "$QUICK_MODE_ACTIVATED" ]]; then
+    pnpm playwright test ./tests/functional/"${TEST_PATHS[@]}" --project=chromium "${SCRIPT_LONG_ARGS[@]}" "${SCRIPT_SHORT_ARGS[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)